### DISCUSSION TITLE
请问如何添加附录的内容？

### DISCUSSION THREAD

**Discussion by shuyangwei** [TheNetAdmin/zjuthesis#415] [General]
没有找到添加附录内容的地方。希望大家能拨冗解答一下！感谢！🙏

**Comment by xfz329** [comment]
本科生 在本科在\body\undergraduate\final\3-appendix.tex

**Reply by xfz329** [reply]
研究生 在 \body\graduate\post\appendix.tex

**Reply by shuyangwei** [reply]
<img width="1078" alt="截屏2024-05-30 16 53 06" src="https://github.com/TheNetAdmin/zjuthesis/assets/99660389/115f80be-9042-4148-b35e-1b7a74b56bec">
找到了，谢谢！！！但是为什么是附录B开始的，我想从附录A开始，被一个图片占去了，怎么删掉那个图片从附录A开始呢？

**Reply by xfz329** [reply]
你修改错了位置，把这个地方的第5行所有语句删除。在\body\graduate\post\appendix.tex 文件中修改附录内容。图片的设定也是在这个地方，删除就好。

**Reply by shuyangwei** [reply]
嗷嗷！找到了！感谢！
